README.md

```diff
@@ -9,10 +9,10 @@
 
 ## What's Golem and yapapi?
 
-**[The Golem Network](https://golem.network)** fosters a global group of creators building ambitious software solutions that will shape the technological landscape of future generations by accessing computing resources across the platform. Golem Network is an accessible, reliable, open access and censorship-resistant protocol, democratizing access to digital resources and connecting users through a flexible, open-source platform.
+**[The Golem Network](https://golem.network)** fosters a global group of creators building ambitious software solutions that will shape the technological landscape of future generations by accessing computing resources across the platform. Golem Network is an accessible, reliable, open and censorship-resistant protocol, democratizing access to digital resources and connecting users in a flexible and open-source manner.
```

Comment:
if I'm interpreting the first sentence correctly - it's the group of creators that shapes the landscape by accessing resources across the platform... sounds somewhat awkward still ... 

maybe it should be "utilizing the" instead of "accessing"?

plus, I'm somehow unsure whether we can use the "across" preposition this way... 

I'd rather say "by utilizing the computing resources available throughout the platform." or just "... available on the platform" ... 

having "utilizing" instead of "accessing" there also removes another repetition...

Reply:
I've updated it now with `utilizing`, I like that we don't have the repetition, but overall I think it's not possible to give an ideal golem description in one or two sentences, without oversimplification or exaggeration.